src/domUpdates.js

```diff
@@ -227,44 +227,53 @@ const updateCurrentRecipe = recipeCard => {
 
 const showGridFeedback = (recipeID, feedback) => {
   const recipe = document.getElementById(recipeID);
-  const gridFeedback = recipe?.querySelector('.grid-feedback');
-  gridFeedback.innerText = feedback;
-  gridFeedback.classList?.add('show-feedback');
+  const gridFeedback = recipe.querySelector('.grid-feedback');
+  gridFeedback?.innerText = feedback;
+  gridFeedback?.classList?.add('show-feedback');
   setTimeout(() => {gridFeedback.classList?.remove('show-feedback')}, 751);
 }
 
+
 const showModalFeedback = (feedback) => {
   let modalFeedback = document.querySelector('.modal-feedback');
-  modalFeedback.innerText = feedback
-  modalFeedback.classList.add('show-feedback');
-  setTimeout(() => {modalFeedback.classList.remove('show-feedback')}, 751)
+  modalFeedback?.innerText = feedback
+  modalFeedback?.classList.add('show-feedback');
+  setTimeout(() => {modalFeedback?.classList.remove('show-feedback')}, 751)
```

Comment:
I get this error when I open up your branch: 
![Screen Shot 2023-06-05 at 8 22 01 AM](https://github.com/lauraguerra1/whats-cookin/assets/121131581/6bd6250a-bb97-4eed-a926-b8ef78cf2035)


I think replacing your question marks with the following conditionals would solve this error and the error you are trying to prevent 

```
const showGridFeedback = (recipeID, feedback) => {
  const recipe = document.getElementById(recipeID);
  if (recipe) {
    const gridFeedback = recipe.querySelector('.grid-feedback');
    gridFeedback.innerText = feedback;
    gridFeedback.classList.add('show-feedback');
    setTimeout(() => {gridFeedback.classList.remove('show-feedback')}, 751);
  }
}


const showModalFeedback = (feedback) => {
  let modalFeedback = document.querySelector('.modal-feedback');
  if (modalFeedback) {
    modalFeedback.innerText = feedback
    modalFeedback.classList.add('show-feedback');
    setTimeout(() => {modalFeedback.classList.remove('show-feedback')}, 751)
  }
}
```

Reply:
done